pkg/provisioners/terraform/compile_test.go

```
@@ -1,21 +1,25 @@
-package terraform
+package terraform_test
 
 import (
 	"bytes"
 	"encoding/json"
 	"os"
 	"path"
 	"testing"
+
+	"github.com/massdriver-cloud/massdriver-cli/pkg/provisioners/terraform"
 )
 
 // Helper function for asserting json serde matches
 func doc(str string) string {
 	b := []byte(str)
 
 	jsonMap := make(map[string](interface{}))
-	json.Unmarshal([]byte(b), &jsonMap)
```

FYI to reviewer this used to make the "empty shema" `doc("")` silently fail. adding this error handling is why I needed to make `doc("{}")` change. 

@chrisghill hopefully this isn't tangibly different in the sentiment of what's being tested. LMK if this needs to be handled differently.

---

Ha! Good catch. This is actually a better check. Terraform rejects an empty json file for tfvars doc:

```
$ terraform plan -var-file lol.tfvars.json
╷
│ Error: Missing value
│
│   on lol.tfvars.json line 1:
│    1:
│
│ The JSON data ends prematurely.
╵
╷
│ Error: Root value must be object
│
│   on lol.tfvars.json line 1:
│    1:
│
│ The root value in a JSON-based configuration must be either a JSON object or a JSON array of objects.
```